client/blocks/importer/wordpress/utils.js

```diff
@@ -43,3 +43,11 @@ export const getAcceptedAssistedFreeMigration = () => {
 		sessionStorage?.getItem( 'wpcom_import_migration_assistance_accepted' )
 	);
 };
+export const setMigrationAssistanceAccepted = () =>
+	ignoreFatalsForSessionStorage( () =>
+		sessionStorage?.setItem( 'wpcom_import_migration_assistance_accepted', true )
+	);
+export const clearMigrationAssistanceAccepted = () =>
```

Comment:
We need to understand when we should clean the session.

Reply:
Yeah, that's true.

Since we're behind a flag, let's tackle this dilemma here: https://github.com/Automattic/wp-calypso/issues/90094